### PR TITLE
Add find tag feature

### PR DESCRIPTION
**Pull request by ElijahS67**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindTagCommand.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.person.PersonContainsTagPredicate;
+
+/**
+ * Lists batchmates that match all tags specified by the user.
+ * Keyword matching is case-sensitive.
+ */
+public class FindTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "findTag";
```
Comment (sltsheryl) [inline src/main/java/seedu/address/logic/commands/FindTagCommand.java:15]: The implementation looks good. 
But based on the user guide, we said to use the command word `sort team4` to find by tag. Which one should we use?

Comment (sltsheryl) [reply src/main/java/seedu/address/logic/commands/FindTagCommand.java:15]: https://ay2223s1-cs2103t-t11-4.github.io/tp/UserGuide.html#sorting-by-tags-sort

Comment (sltsheryl) [reply src/main/java/seedu/address/logic/commands/FindTagCommand.java:15]: Both are fine, but we either have to change it here or the user guide. 
Personally I think findTag is more intuitive too.

Comment (carriezhengjr) [reply src/main/java/seedu/address/logic/commands/FindTagCommand.java:15]: By the way, I'm updating the user guide. So it doesn't necessarily have to be `sort`; depends on which one we prefer, `sort` or `findTag`. 😃

Comment (ElijahS67) [reply src/main/java/seedu/address/logic/commands/FindTagCommand.java:15]: I renamed cos `sort` sounds kind of weird if we are finding by tag though

**Pull request merged**